cmd/statusd/main.go

```diff
@@ -85,7 +85,7 @@ var (
 	// LogLevelFlag defines a log reporting level
 	LogLevelFlag = cli.StringFlag{
 		Name:  "log",
-		Usage: `Log level, one of: "ERROR", "WARN", "INFO", "DEBUG", and "TRACE"`,
+		Usage: `Log level, one of: "ERROR", "YELLOWALERT", "INFO", "REDALERT"`,
```

Comment:
@tiabc  @divan I believe this guys suite more for the purpose we want both from discussions from #295 and the fact that i believe `Trace` should naturally be part of log entries, where each has `file` and `line number` associated with area of called function.